Client/YooChooseNotifier.php

```diff
@@ -136,7 +134,16 @@ protected function notify(array $events)
 
         $response = $this->guzzle->post(
             $this->getNotificationEndpoint(),
-            array( 'json' => array( 'transaction' => null, 'events' => $events ) )
+            array(
+                'json' => array(
+                    'transaction' => null,
+                    'events' => $events
+                ),
+                'auth' => array(
```

Comment:
[it works and I don't know why](http://img-9gag-fun.9cache.com/photo/anYZ9Eo_700b.jpg)